x-pack/plugins/security_solution/public/common/components/event_details/get_alert_summary_rows.tsx

```diff
@@ -151,50 +129,34 @@ export const getSummaryRows = ({
   const tableFields = getEventFieldsToDisplay({ eventCategory, eventCode });
 
   return data != null
-    ? tableFields.reduce<SummaryRow[]>((acc, item) => {
-        const initialDescription = {
-          contextId: timelineId,
-          eventId,
-          isDraggable,
-          value: null,
-          fieldType: 'string',
-          linkValue: undefined,
-          timelineId,
-        };
-        const field = data.find((d) => d.field === item.id);
-        if (!field || isEmpty(field?.values)) {
+    ? tableFields.reduce<SummaryRow[]>((acc, field) => {
+        const item = data.find((d) => d.field === field.id);
+        if (!item || isEmpty(item?.values)) {
           return acc;
         }
 
         const linkValueField =
-          item.linkField != null && data.find((d) => d.field === item.linkField);
-        const linkValue = getOr(null, 'originalValue.0', linkValueField);
-        const value = getOr(null, 'originalValue.0', field);
-        const category = field.category ?? '';
-        const fieldName = field.field ?? '';
-
-        const browserField = get([category, 'fields', fieldName], browserFields);
+          field.linkField != null && data.find((d) => d.field === field.linkField);
         const description = {
-          ...initialDescription,
-          data: {
-            field: field.field,
-            format: browserField?.format ?? '',
-            type: browserField?.type ?? '',
-            isObjectArray: field.isObjectArray,
-            ...(item.overrideField ? { field: item.overrideField } : {}),
-          },
-          values: field.values,
-          linkValue: linkValue ?? undefined,
-          fieldFromBrowserField: browserField,
+          ...getEnrichedFieldInfo({
+            item,
+            linkValueField: linkValueField || undefined,
```

Comment:
nit: just curious if the switch from `??` to `||` when `linkValue: linkValue ?? undefined` became `linkValueField: linkValueField || undefined,` was intentional

Reply:
Jep, I want to make sure it's actually undefined if it's falsy